### PR TITLE
188 - Expand /status endpoint with more information

### PR DESCRIPTION
**Pull request by LukasKri**
Workshop task.

### CODE REVIEW

File: src/getVersion.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import fs from 'fs';
+
+export default async function getVersion() {
+    const packageJson = await fs.promises.readFile('./package.json', 'utf8');
```
Comment (psaulenas) [inline src/getVersion.ts:4]: You could get the version using import from `package.json`
```js
import { version } from '../package.json';
```

Then probably this separate file (function) not needed

Comment (LukasKri) [reply src/getVersion.ts]: Oh really, that's so much simpler! I updated my code according to the example provided and squashed my commits. Thanks @psaulenas!

**Comment by ignasposka** [comment]
Sorry, but this feature was merged from another PR.

**Comment by ignasposka** [comment]
Sorry, but this feature was merged from another PR.

**Comment by ignasposka** [comment]
Sorry, but this feature was merged from another PR.

**Pull request closed without merging**